tests/lax_scipy_test.py

```diff
@@ -312,6 +314,116 @@ def apply_normalization(a):
       lsp_special_fn = lambda z: lsp_special.lpmn_values(l_max, l_max, z, is_normalized)
       self._CompileAndCheck(lsp_special_fn, args_maker)
 
+  def testSphHarmAccuracy(self):
+    m = jnp.array([-10, -2, 0, 1, 2])
+    n = jnp.array([20, 3, 1, 2, 6])
+    n_max = 20
+    theta = jnp.array([0.1, 0.1, 0.3, 0.5, 0.9])
+    phi = jnp.array([0.2, 0.2, 0.5, 1.0, 0.2])
+
+    expected = lsp_special.sph_harm(m, n, theta, phi, n_max)
+
+    actual = osp_special.sph_harm(m, n, theta, phi)
+
+    with self.subTest('Test real part.'):
+      self.assertAllClose(
+          jnp.real(actual), jnp.real(expected), rtol=1e-8, atol=9e-5)
+
+    with self.subTest('Test imaginary part.'):
+      self.assertAllClose(
+          jnp.imag(actual), jnp.imag(expected), rtol=1e-8, atol=9e-5)
```

Comment:
I believe that `self.assertAllClose` should be able to compare complex values directly, rather than comparing the real & imag parts separately.

Reply:
Done.